TDAT: protected void set up once ( test result result ) {  <NL> for ( iterator i = get tests ( ) . iterator ( ) ; i . has next ( ) ; ) {  <NL> test test = ( test ) i . next ( ) ;  <NL> if ( test instanceof j func test case ) {  <NL> try {  <NL>  ( ( j func test case ) test ) . set up once ( ) ;  <NL>  } catch ( exception e ) {  <NL> result . add error ( test , e ) ;  <NL>  }  <NL>  }  <NL>  }  <NL>  }  COM: <s> this doesnt really work the way i want it to just yet </s>